### ISSUE TITLE
lab run results - 20161025

### ISSUE THREAD

**Issue by jduncan-rva** [RedHatGov/ansible-workshop#5]
I just ran through an abbreviated version of the lab, and there were a few issues that popped up after deploying the gold image.
- we had some connectivity issues, but they turned out to be the DNS servers at the site being overloaded. Nothing we could do about that.
- setting mass permissions (known issue)
- cloud credential - the gold image doesn't have one configured. we should figure that out.
- job templates - for example the motd template, when i went to look at it, it didn't have anything actually configured. it wasn't associated with a project. which makes since since i bootstrapped the project, but the job templates were in the image already.

instead of a gold image i think we need to work out a soup>nuts deployment/bootstrap playbook.

**Comment by bhirsch70** [comment]
Agree.  The gold image idea could be irrelevant if we enhance the bootstrap to not only configure tower for the workshop, but also install tower in the first place.  In addition, this could perform the AWS instance build for the tower server itself.  I believe the BD team did some of this already, so we should get their stuff in here.  I know this has already been requested.
